test/fuzz/generateKey.js

```diff
@@ -0,0 +1,26 @@
+import { FuzzedDataProvider } from '@jazzer.js/core';
+
+import openpgp from '../initOpenpgp.js';
+
+const MAX_NAME_LENGTH = 30;
+const MAX_COMMENT_LENGTH = 500;
+
+/**
+ * @param { Buffer } inputData
+ */
+export function fuzz (inputData) {
+
+  const data = new FuzzedDataProvider(inputData);
+  const asciiString = data.consumeString(MAX_COMMENT_LENGTH);
+  const utf8String = data.consumeString(MAX_NAME_LENGTH, 'utf-8');
+
+  return openpgp.generateKey({ userIDs: [
```

Comment:
It seems to me this is testing the UserID parsing part more than the key generation itself (which happens to be a not-so-cheap function to run)?
For the same purpose, I'd directly fuzz `UserID.fromObject()`.

I'm not sure we need to fuzz `generateKey` in general, for similar reasons as the `create*` methods (cc @twiss).

Reply:
I've added a fuzz test for `UserIDPacket.fromObject()`